Mathlib/Topology/Algebra/Valued/ValuedField.lean

```diff
@@ -362,3 +362,38 @@ theorem valuedCompletion_apply (x : K) : Valued.v (x : hat K) = v x :=
 #align valued.valued_completion_apply Valued.valuedCompletion_apply
 
 end Valued
+
+section Notation
+
+namespace Valued
+
+/-- A `Valued` version of `Valuation.integer`, enabling the notation `𝒪[K]` for the
+valuation integers of a valued field `K`. -/
+@[reducible]
+def integer (K : Type*) [Field K] {Γ : outParam Type*} [LinearOrderedCommGroupWithZero Γ]
```

Comment:
Please consider extracting these variables into a `variable` statement.

Reply:
Thank you! Fixed.